### PR TITLE
Changes on Cookie and Privacy Policy

### PR DESCRIPTION
**Pull request by MartinRst**
modification on cookies and Privacy Pages

### CODE REVIEW

File: content/cookie-policy.md [hunk 1]
```diff
@@ -4,73 +4,50 @@ description: Learn about our cookie policy, detailing how we use cookies to enha
 ---
 
 # Cookie Policy
+-----------------
 
-This Cookies Policy explains what Cookies are and how We use them. You should read this policy so You can understand what type of cookies We use, or the information We collect using Cookies and how that information is used.
+Last updated: 4th of May 2023
 
-Cookies do not typically contain any information that personally identifies a user, but personal information that we store about You may be linked to the information stored in and obtained from Cookies. For further information on how We use, store and keep your personal data secure, see our Privacy Policy.
+This Cookie Policy explains how Kestra uses cookies on our website [kestra.io](http://kestra.io) (the "Website"). By using our Website, you consent to the use of cookies in accordance with this Cookie Policy.
```
Comment (anna-geller) [inline content/cookie-policy.md:11]:
```suggestion
This page explains how Kestra uses cookies on [kestra.io](http://kestra.io) (i.e., the "Website"). By using the website, you consent to the use of cookies in accordance with this cookie policy.
```

Comment (anna-geller) [reply content/cookie-policy.md:11]: I would generally make all those lowercase: website, cookie policy -- but up to you. The main thing would be to keep it consistent. Atm, the website is sometimes named in lowercase and sometimes capitalized (Website).

File: content/cookie-policy.md [hunk 2]
```diff
@@ -4,73 +4,50 @@ description: Learn about our cookie policy, detailing how we use cookies to enha
...
 
-Cookies can be "Persistent" or "Session" Cookies. Persistent Cookies remain on your personal computer or mobile device when You go offline, while Session Cookies are deleted as soon as You close your web browser.
+We use the following third-party cookies on our Website:
 
-We use both session and persistent Cookies for the purposes set out below:
+1.  **Google Analytics**: We use Google Analytics to collect information about how visitors use our Website, such as the number of visitors, the pages they visit, and the duration of their visit. This information is used to improve the user experience and optimize our Website. You can learn more about Google Analytics cookies and how to opt-out **[here](https://tools.google.com/dlpage/gaoptout)**.
+2.  **HubSpot**: We use HubSpot to manage our customer relationships and provide a better user experience on our Website. HubSpot uses cookies to track visitor activity, gather data about website usage, and understand user preferences. You can learn more about HubSpot cookies and how to opt-out **[here](https://knowledge.hubspot.com/reports/what-cookies-does-hubspot-set-in-a-visitor-s-browser)**.
```
Comment (anna-geller) [inline content/cookie-policy.md:32]:
```suggestion
2.  **HubSpot**: We use HubSpot to manage our customer relationships and provide a better user experience on our Website. HubSpot uses cookies to track visitor activity, gather data about website usage, and understand user preferences. You can learn more about HubSpot cookies and how to opt out of using them **[here](https://knowledge.hubspot.com/reports/what-cookies-does-hubspot-set-in-a-visitor-s-browser)**.
```

**Review by anna-geller — APPROVED** [review]
LGTM - requested only small adjustments for consistency

File: content/cookie-policy.md (hunk 1 above)
Comment (MartinRst) [reply content/cookie-policy.md:11]: yeah I stuttered on the MD!

**Pull request merged**